recipe/meta.yaml

```diff
@@ -7,16 +7,16 @@ source:
     git_tag: v4.4.3
 
 build:
-    number: 1
+    number: 2
     skip: True  # [win]
 
 requirements:
     build:
-        - libnetcdf
+        - libnetcdf 4.3*
         - krb5
```

Comment:
I have no idea what changed in the default channel that forced us to install `krb5` everywhere. I guess we will only find out if we start removing it and checking what breaks.